src/main/kotlin/no/nav/syfo/personstatus/infrastructure/database/repository/PersonOversiktStatusRepository.kt

```diff
@@ -40,7 +41,34 @@ class PersonOversiktStatusRepository(private val database: DatabaseInterface) :
                     Result.success(rowsUpdated)
                 } else {
                     connection.rollback()
-                    Result.failure(RuntimeException("Failed to update arbeidsuforhet vurdering status for person with fnr: ${personident.value}"))
+                    Result.failure(RuntimeException("Failed to update arbeidsuforhet vurdering status for personstatus: $uuid"))
+                }
+            }
+        } catch (e: Exception) {
+            Result.failure(e)
+        }
+    }
+
+    override fun upsertMerOppfolgingStatus(personident: PersonIdent, isUbehandletKandidat: Boolean): Result<Int> {
+        return try {
+            database.connection.use { connection ->
+                val now = Timestamp.from(Instant.now())
+                val uuid = UUID.randomUUID().toString()
+                val rowsUpdated = connection.prepareStatement(UPSERT_PERSON_OVERSIKT_STATUS_MER_OPPFOLGING).use {
+                    it.setString(1, uuid)
+                    it.setString(2, personident.value)
+                    it.setBoolean(3, isUbehandletKandidat)
+                    it.setTimestamp(4, now)
+                    it.setTimestamp(5, now)
+                    it.executeUpdate()
+                }
+                val isSuccess = rowsUpdated == 1
+                if (isSuccess) {
+                    connection.commit()
+                    Result.success(rowsUpdated) // TODO: Gir det mening å returnere antall rader som er oppdatert?
```

Comment:
Gir det tallet oss noe? Ville heller tenkt at man ga rad-IDen fra databasen. Eller er det bare for å ha noe å returnere `Result` med?